hypervisor/qemu/qemu_process.go

```diff
@@ -13,6 +14,47 @@ import (
 	"github.com/hyperhq/runv/hypervisor"
 )
 
+// implement io.Reader interface
+type qemuLogReader struct {
+	logFile string
+	offset  int64
+}
+
+func (f *qemuLogReader) Read(p []byte) (n int, err error) {
+	reader, err := os.Open(f.logFile)
+	defer reader.Close()
+	if err != nil {
+		return 0, err
+	}
+	reader.Seek(f.offset, 0)
```

Comment:
Support "the reopen-seek until meet io.EOF"  when implements createQemuLogReader.